damnit/migrations.py

```diff
@@ -199,10 +199,24 @@ def migrate_v0_to_v1(db, db_dir, dry_run):
     new_db_path = db_dir / "runs.v1.sqlite"
     new_db_path.unlink(missing_ok=True)  # Clear any previous attempt
     new_db = DamnitDB(new_db_path)
+
+    # Copy the metadata
     for k, v in db.metameta.items():
         if k != "data_format_version":
             new_db.metameta[k] = v
 
+    # Copy the user-editable variables and standalone comments
+    for table in ["variables", "time_comments"]:
+        rows = [rec for rec in db.conn.execute(f"SELECT * FROM {table}").fetchall()]
```

Comment:
```suggestion
        rows = db.conn.execute(f"SELECT * FROM {table}").fetchall()
```

The list comprehension is redundant here - `.fetchall()` already gives you a list :wink:

Reply:
Ack, I didn't read properly :facepalm: Fixed in 7d832ad.